site/src/components/GlobalSnackbar/GlobalSnackbar.tsx

```diff
@@ -29,54 +28,10 @@ export const GlobalSnackbar: FC = () => {
   const [open, setOpen] = useState<boolean>(false);
   const [notification, setNotification] = useState<NotificationMsg>();
 
-  const handleNotification = useCallback<CustomEventListener<NotificationMsg>>(
-    (event) => {
-      setNotification(event.detail);
-      setOpen(true);
-    },
-    [],
-  );
-
-  useCustomEvent(SnackbarEventType, handleNotification);
-
-  const renderAdditionalMessage = (msg: AdditionalMessage, idx: number) => {
```

Comment:
This got removed in favor of making it a separate component at the bottom of the file